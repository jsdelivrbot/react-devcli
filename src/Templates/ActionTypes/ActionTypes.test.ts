import {js, ts} from './ActionTypes';

describe('ActionTypes template test suite', (): void => {
    test('should match action types template snapshot for JS language', (): void => {
        expect(js()).toMatchSnapshot();
    });

    test('should match action types template snapshot for TS language', (): void => {
        expect(ts()).toMatchSnapshot();
    });
});
