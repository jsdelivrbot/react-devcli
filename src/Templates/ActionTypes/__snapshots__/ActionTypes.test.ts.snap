// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionTypes template test suite should match action types template snapshot for JS language 1`] = `""`;

exports[`ActionTypes template test suite should match action types template snapshot for TS language 1`] = `""`;
